src/views/AddItem.jsx

```diff
@@ -3,6 +3,9 @@ import { addItem } from '../api/firebase';
 import { RoughNotation } from 'react-rough-notation';
 import { UrgencyTag } from '../components/UrgencyTag';
 
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPenToSquare as pen } from '@fortawesome/free-regular-svg-icons';
```

Comment:
I'm wondering if this is the right icon?!? Maybe we should use the same one as the NavBar for addItem